RELEASE-NOTES.txt

```diff
@@ -2,6 +2,10 @@
 
 16.8
 -----
+* [*] Block Editor: Fixed block mover title wording for better clarity from 'Move block position' to 'Change block position'. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3049]
+* [**] Block Editor: Make inserter long-press options "add to beginning" and "add to end" always available. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3074]
```

Comment:
This change was included in the 1.46.1 beta fix so it should better go under 16.7 section.